tests/solids/diamondC_1x1x1_pp/CMakeLists.txt

```diff
@@ -74,35 +74,44 @@ qmc_run_and_check(
   DIAMOND_SCALARS # VMC
 )
 
-if(NOT ENABLE_OFFLOAD)
-  list(APPEND DIAMOND_KSPACE_SCALARS "totenergy" "-10.500719 0.001769")
-  list(APPEND DIAMOND_KSPACE_SCALARS "variance" "0.312264 0.028662")
-  qmc_run_and_check(
-    short-diamondC_1x1x1_pp-vmc_sdj_kspace
-    "${qmcpack_SOURCE_DIR}/tests/solids/diamondC_1x1x1_pp"
-    qmc_short_kspace
-    qmc_short_kspace.in.xml
-    1
-    16
-    TRUE
-    0
-    DIAMOND_KSPACE_SCALARS # VMC
-  )
+list(APPEND DIAMOND_KSPACE_SCALARS "totenergy" "-10.500719 0.001769")
+list(APPEND DIAMOND_KSPACE_SCALARS "variance" "0.312264 0.028662")
+qmc_run_and_check(
+  short-diamondC_1x1x1_pp-vmc_sdj_kspace
+  "${qmcpack_SOURCE_DIR}/tests/solids/diamondC_1x1x1_pp"
+  qmc_short_kspace
+  qmc_short_kspace.in.xml
+  1
+  16
+  TRUE
+  0
+  DIAMOND_KSPACE_SCALARS # VMC
+)
+
+qmc_run_and_check(
+  short-diamondC_1x1x1_pp-vmc_sdj_kspace
+  "${qmcpack_SOURCE_DIR}/tests/solids/diamondC_1x1x1_pp"
+  qmc_short_kspace_4_4
+  qmc_short_kspace_4_4.in.xml
+  4
+  4
```

Comment:
This test seems being added legacy only. If we only intend to have this test either legacy or batch, I think batch is preferred.

Reply:
The batched driver test is added at line 103. The ones above are not new, except I changed them to always run. Since the tests are fast, I think we should keep both tests to reduce the chance for problems.

Reply:
I see. Do you think it is necessary to add this case to batched as well?

Reply:
I can if you would like it added. Since the Jastrows involve no parallelism, I generally don't think they add much. Your call.

Reply:
Please do it in this PR and we won't rediscover missing coverage.

Reply:
Added. Also added samples check.